features/bevragen/persoon/partner/in-onderzoek.feature

```diff
@@ -0,0 +1,53 @@
+# language: nl
+
+Functionaliteit: Persoon: partner velden zijn in onderzoek
+
+  Abstract Scenario: '<type>' is in onderzoek en alle 'root' velden wordt gevraagd
```

Comment:
Bedoel je dat er hier drie "root" gegevens als voorbeeld worden gevraagd van de 8? Of heb ik nu geen goed beeld van wat met "root" wordt bedoeld?

Reply:
de 'root' velden zijn de velden op het hoogste niveau van een component. Ik denk dat dit moeilijk is te begrijpen als je niet programmeert. Ik schrijf het wel uit